flutter_vlc_player/lib/src/vlc_player_controller.dart

```diff
@@ -97,6 +97,17 @@ class VlcPlayerController extends ValueNotifier<VlcPlayerValue> {
   /// Initialize vlc player when the platform is ready automatically
   final bool autoInitialize;
 
+  /// This parameter is deprecated, please, use the [addOnInitListener] method instead.
+  /// This is a callback that will be executed once the platform view has been initialized.
+  /// If you want the media to play as soon as the platform view has initialized, you could just call
+  /// [VlcPlayerController.play] in this callback. (see the example)
```

Comment:
```suggestion
  /// This is a callback that will be executed once the platform view has been initialized.
  /// If you want the media to play as soon as the platform view has initialized, you could just call
  /// [VlcPlayerController.play] in this callback. (see the example).
  ///
  /// This member is deprecated, please, use the [addOnInitListener] method instead.
```
What do you think about moving the deprecation message to the documentation end? Also, should we mark this field as `deprecated` too?

Reply:
i moved that line to the end of documentation.
about `deprecated` keyword, because they're private callbacks i don't think it's necessary to mark them.